packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -736,7 +736,7 @@ -(void)copyToClipboard:(NSString *)content {
 - (void)setDevMenuAppBridge
 {
   DevMenuManager *manager = [DevMenuManager shared];
-  manager.currentBridge = self.appBridge;
+  manager.currentBridge = [RCTBridge currentBridge];
```

Comment:
is that intentional? if `appBridge` is not being used, could we remove them?
also kinda afraid of the loading sequence between app bridge and dev launcher bridge. we cannot ensure RCTBridge.currentBridge is app bridge or EXDevLauncherRCTBridge.

Reply:
Reverted this back to `self.appBridge`